lib/cre8or.ts

```diff
@@ -24,3 +27,21 @@ export const getLockedCount = async (address: string) => {
 
   return count
 }
+
+export const transferCre8orToSmartWallet = async (
+  contractAddress: string,
+  from: string,
+  to: string,
+  tokenId: number,
+  signer: Signer,
+) => {
+  try {
+    const contract = new ethers.Contract(contractAddress, cre8orAbi, signer)
```

Comment:
import `Contract` directly from `ethers` for cleaner code ✨ 
```suggestion
    const contract = new Contract(contractAddress, cre8orAbi, signer)
```

Reply:
fixed 👍